medical/models/medical_abstract_entity.py

```diff
@@ -195,14 +203,13 @@ def _get_default_image(self, vals):
                     'base', 'static/src/img', 'patient-avatar.png',
                 )
                 with open(img_path, 'r') as image:
-                    base64_image = image.read().encode('base64')
-                    return odoo.tools.image_resize_image_big(base64_image)
+                    return image
 
         Args:
             vals (dict): Values dict as passed to create.
 
         Returns:
-            str: Base64 encoded image if there was one.
+            file: A file-like object representing the image.
             bool: False if error.
             NoneType: None if no result.
         """
```

Comment:
Where-ever you store I think it will be the same work: override the function `_get_default_image`